tests/Vimeo/VimeoTest.php

```diff
@@ -176,7 +176,7 @@ public function testUploadWithFakeMachineIdShouldReturnVimeoRequestException()
         $vimeo = new Vimeo($this->clientId, $this->clientSecret);
 
         // Act
-        $result = $vimeo->upload(__DIR__.'/../../composer.json');
+        $result = $vimeo->upload(__DIR__.'/../../composer.json', array('fake_machine_id'));
```

Comment:
We don't support machine ID's anymore in the API, so you can remove these changes to `VimeoTest`.